analyses/fusion_filtering/03-Calc-zscore-annotate.R

```diff
@@ -144,52 +99,111 @@ ZscoredAnnotation<-function(standardFusionCalls=standardFusionCalls,zscoreFilter
     dplyr::arrange(Sample, FusionName) %>%
     # Retain only distinct rows
     dplyr::distinct()
-
- # check columns for Gene1A,Gene2A,Gene1B and Gene2B
- cols<-c(Gene1A=NA, Gene1B=NA, Gene2A=NA, Gene2B=NA)
-
-  expression_annotated_fusions <- fusion_sample_gene_df %>%
-    # join the filtered expression values to the data frame keeping track of symbols
-    # for each sample-fusion name pair
-    dplyr::left_join(expression_long_df, by = c("Sample", "GeneSymbol")) %>%
-    # for each sample-fusion name pair, are all genes under the expression threshold?
-    dplyr::group_by(FusionName, Sample) %>%
-    dplyr::select(FusionName, Sample,zscore_value,gene_position) %>%
-    # cast to keep zscore and gene position
-    reshape2::dcast(FusionName+Sample ~gene_position,value.var = "zscore_value") %>%
-    # incase Gene2A/B dont exist like in STARfusion calls
-    add_column(!!!cols[setdiff(names(cols),names(.))]) %>%
-    # get annotation from z score
-    dplyr::mutate(note_expression_Gene1A = ifelse((Gene1A>zscoreFilter| Gene1A< -zscoreFilter),"differentially expressed","no change"),
-                  note_expression_Gene1B = ifelse((Gene1B>zscoreFilter| Gene1B< -zscoreFilter),"differentially expressed","no change"),
-                  note_expression_Gene2A = ifelse((Gene2A>zscoreFilter| Gene2A< -zscoreFilter),"differentially expressed","no change"),
-                  note_expression_Gene2B = ifelse((Gene2B>zscoreFilter| Gene2B< -zscoreFilter),"differentially expressed","no change")) %>%
-    dplyr::rename(zscore_Gene1A=Gene1A,
-                  zscore_Gene1B=Gene1B,
-                  zscore_Gene2A=Gene2A,
-                  zscore_Gene2B=Gene2B) %>%
-    # unique FusionName-Sample rows
-    # use this to filter the QC filtered fusion data frame
-    dplyr::inner_join(standardFusionCalls, by = c("FusionName", "Sample")) %>%
-    dplyr::distinct()
-
-  return (expression_annotated_fusions)
+  
+  expression_annotated_cohort_combined <- data.frame()
+  for (j in 1:length(cohortInterest)){
+    cohort_name = cohortInterest[j]
+    
+    # example run using GTEx
+    # load GTEx data - make sure the expression data matches
+    normData<-readRDS(gtexMatrix[j]) 
```

Comment:
`gtexMatrix` is not an input to the function can you update this to `normData` with the above suggested change?